### ISSUE TITLE
cursor stuck on image result in default "all" section when google suggests images on firefox.

### ISSUE THREAD

**Issue by userrand** [infokiller/web-search-navigator#486]
Thank you for this nice extension.

Copy-pasted the title:

"cursor stuck on image result in default "all" section when google suggests images on firefox."

pressing up or down does not exit the embedded image result/section

search query example : spider diagram examples

![image](https://user-images.githubusercontent.com/114431098/222695305-a9988fb0-9259-4708-bac6-2d75ce6df42a.png)

**Comment by infokiller** [comment]
Can't reproduce it, please follow the steps here and report back: https://github.com/infokiller/web-search-navigator#troubleshooting

**Comment by userrand** [comment]
Thank you for your quick response.

I turned off extensions in chrome and had the same issue when searching for the same query: spider diagram examples

However,

when I searched on firefox for: anki site:github.com

I was able to go through the website results by pressing down although the first answer was also a list of images.

I checked the image section using inspect on the browser and did not notice a difference in the section title but I did not check the environments of the two sections. 

In any case, it seems to not always happen so it's not too much of a problem.

Thank you again for your nice extension and your help.

**Comment by infokiller** [comment]
Thanks for the update and reporting the issue, I'll close the issue for now but if you find a reliable way to reproduce it, please let me know, I do want to fix issues that I can reproduce

**Comment by userrand** [comment]
another case where the navigator gets stuck due to goggle image results (on edge browser):

searching for the color blue :

https://www.google.com/search?q=blue

**Comment by userrand** [comment]
I found out think I found out what the issue is. 

When I un-zoom the extension is able to scroll through the search results. Maybe there is something in the code that uses the viewport of the user such that it would get stuck if the next element the next search result isn't within the viewport. This tends to happen when the search result has images or google provides related results below the result.
